tst/EnergyPlus/unit/InputProcessor.unit.cc

```diff
@@ -623,6 +623,39 @@ TEST_F(InputProcessorFixture, parse_bad_utf_8_json_3)
     EXPECT_EQ( expected, input_file );
 }
 
+TEST_F(InputProcessorFixture, parse_latin1_json)
+{
+
+    // Test for #7388 -  Non UTF-8 characters throwing false blank name error
+
+    std::string const idf(delimited_string({
+        "  Construction,",
+        "    \x31\xB0\x70\x69\x61\x6E\x6F,  !- Name",
```

Comment:
Did someone let Elon Musk name this construction object?

Reply:
I'm just faking a latin1 encoding here. In the Defect file, the only "problem" is the use of `°`  and the fact that the file is latin-1 encoded (well, a derivative, it's probably a Windows-1252 if I were to guess)